pkg/networkgraph/tree/types.go

```diff
@@ -14,6 +16,7 @@ type NetworkTree interface {
 	// is equal to the cardinality. If there are multiple trees, the checks are done
 	// for each tree.
 	ValidateNetworkTree() bool
+	findCIDRNoLock(ipNet *net.IPNet) (*nRadixNode, error)
```

Comment:
This is needed to test `findCIDRNoLock`. Is the unit test worth making this change?

Reply:
An alternative way of handling this would be to create a test, where the `NewNRadixTree` constructor is not used, so that we can create an `nRadixTree` object and call `findCIDRNoLock` on it directly.

Reply:
I went with your second suggestion.